src/patternfly/components/Chip/chip.scss

```diff
@@ -30,6 +35,9 @@
   // chip badge
   --pf-c-chip__c-badge--MarginLeft: var(--pf-global--spacer--xs);
 
+  // icon
+  --pf-c-chip__icon--MarginLeft: var(--pf-global--spacer--xs);
```

Comment:
This spacing looks tighter than the other examples. Should it be `--pf-c-chip__icon--MarginLeft: var(--pf-global--spacer--sm);`